packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Button, Modal, DesktopWrapper, MobileDialog, MobileWrapper, UILoader, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer } from 'mobx-react-lite';
+import classNames from 'classnames';
+import { useStores } from 'Stores/index';
+import TradigPlatformIconProps from 'Assets/svgs/trading-platform';
+import { TIconTypes } from 'Types';
+
+type TModalContent = {
+    account_type_card: string;
+    selectAccountTypeCard: React.Dispatch<React.SetStateAction<string>>;
+};
+
+type TAccountType = {
+    title_and_type: string;
+    description: string;
+    icon: string;
+};
+
+const account_types: TAccountType[] = [
+    {
+        title_and_type: 'Derived',
```

Comment:
I believe `CFD` is missing from this. We need to cater for MF clients as well.
Also we need to get `trading_platform_available_accounts` from the API and map them to the details from an array.